great_expectations/self_check/util.py

```diff
@@ -938,14 +938,9 @@ def locale_wrapper(*args: P.args, **kwargs: P.kwargs) -> None:
         old_locale = locale.setlocale(locale.LC_TIME, None)
         print(old_locale)
         # old_locale = locale.getlocale(locale.LC_TIME) Why not getlocale? not sure
-        try:
-            new_locale = locale.setlocale(locale.LC_TIME, "en_US.UTF-8")
-            assert new_locale == "en_US.UTF-8"
-            func(*args, **kwargs)
-        except Exception:  # noqa: TRY203
-            raise
-        finally:
-            locale.setlocale(locale.LC_TIME, old_locale)
+        new_locale = locale.setlocale(locale.LC_TIME, "en_US.UTF-8")
```

Comment:
You've removed the finally here. It seems like we need it to restore the correct locale?